integration-tests/cli/two-source/test.test.js

```diff
@@ -0,0 +1,10 @@
+const test = require('ava');
+const {exec} = require('../../setup');
+
+test.failing(
+    'parser: able to import variable from module',
```

Comment:
Perhaps let's not mark it as `test.failing` ("meant to fail") but `test` (meant to work, and failures should be reported)?

Also I'd write something more in the message about there being two source directories, given that's in the test folder name and that's what the Elm source files are actually doing.

Reply:
```suggestion
    'cli: supports multiple source directories',
```